src/app/applications/multicast/multicast.service.ts

```diff
@@ -88,8 +81,22 @@ export class MulticastService {
       updatedByName: this.userMinimalService.getUserNameFrom(
         multicastResponse.updatedBy
       ),
-      
     };
     return model;
   }
+
+  public multicastGet(multicastId: number, params = {}): Observable<any> {
```

Comment:
Let's avoid introducing more `any` than necessary. Use the `unknown` type when possible to force developers to assert the type before acting on it.
[Documentation](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type)